packages/react-redux-subspace/test/components/SubspaceProvider-spec.js

```diff
@@ -37,6 +37,74 @@ describe('SubspaceProvider Tests', () => {
 
         expect(testComponent.text()).to.equal("expected")
     })
+
+    it('should be able to provide a subspace on an alternate context key', () => {
+        let state = {
+            subState: {
+                value: "expected"
+            },
+            value: "wrong"
+        }
+
+        let mockStore = configureStore()(state)
+
+        let storeKey = 'subStateKey'
+
+        let AlternateContextKeyTestComponent = connect(
+            state => { return { value: state.value } },
+            null,
+            null,
+            {storeKey}
+        )(props => <p>{props.value}</p>)
+
+        let Provider = createProvider(storeKey)
+        let AlternateContextKeySubspaceProvider = createSubspaceProvider(storeKey)
+
+        let testComponent = render(
+            <Provider store={mockStore}>
+                <AlternateContextKeySubspaceProvider mapState={state => state.subState}>
+                    <AlternateContextKeyTestComponent />
+                </AlternateContextKeySubspaceProvider>
+            </Provider>
+        )
+
+        expect(testComponent.text()).to.equal("expected")
+    })
+
+    it('should be able to consume a redux store defined on an alternate context key', () => {
+        let state = {
+            subState: {
+                value: "expected"
+            },
+            value: "wrong"
+        }
+
+        let mockStore = configureStore()(state)
+
+        let parentStoreKey = 'parentStoreKey'
+        let storeKey = 'storeKey'
+
+        let Provider = createProvider(parentStoreKey)
+
+        let AlternateContextKeyTestComponent = connect(
+            state => { return { value: state.value } },
+            null,
+            null,
+            {storeKey}
+        )(props => <p>{props.value}</p>)
+
+        let AlternateContextKeySubspaceProvider = createSubspaceProvider({storeKey, parentStoreKey})
+
+        let testComponent = render(
+            <Provider store={mockStore}>
+                <AlternateContextKeySubspaceProvider mapState={state => state.subState}>
+                    <AlternateContextKeyTestComponent />
+                </AlternateContextKeySubspaceProvider>
+            </Provider>
+        )
+
+        expect(testComponent.text()).to.equal("expected")
+    })
         
```

Comment:
wouldn't mind also seeing some cases for the defaulting of the keys when only one is provided, i.e.

* `createSubspaceProvider({ childStoreKey })`
* `createSubspaceProvider({ parentStoreKey })`